front_end/src/components/layout/Navbar.js

```diff
@@ -6,30 +6,20 @@ import { logoutUser } from "../../actions/authActions";
 import { submitQuery } from "../../actions/queryActions";
 import { clearCurrentProfile } from "../../actions/profileActions";
 import AnchorLink from "react-anchor-link-smooth-scroll";
+import NavbarMenu from "./NavbarMenu";
+
+import "./navbar.css";
 
 // Material-UI Imports Below
-import {
-  AppBar,
-  Toolbar,
-  Typography,
-  Button,
-  withStyles,
-  Grid,
-  IconButton
-} from "@material-ui/core";
+import { Button, withStyles, Grid } from "@material-ui/core";
```

Comment:
please remove withStyles at import and export down below if you are not using it

Reply:
done